content/6_Advanced/XOR_Basis.mdx

```diff
@@ -27,6 +26,244 @@ frequency: 1
 	</Resource>
 </Resources>
 
+## Introduction
```

Comment:
we should probably add some prereq stuff about linear algebra first

Reply:
I don't think so, the basics of XOR basis is just bit manipulation.

Reply:
Do you want me to add this link in prereq
https://cp-algorithms.com/linear_algebra/linear-system-gauss.html

Reply:
i was thinking moreso a list of linalg topics people are expected to understand before going into this module
feel like that's better than a half-baked explanation on what bases, LI-ness is and whatnot

Reply:
It would be helpful if you like suggest some of the topics

Reply:
Or .... since it is one of the advanced topics we can expect people to be familiar with Linear Algebra. The least I can do is like include the CP algorithms blog about Linear Algebra.

Reply:
no yeah but like at least we should add a disclaimer that "hey, this topic does use some funny math stuff"

Reply:
I know that's why I am can I add the this link: https://cp-algorithms.com/linear_algebra/linear-system-gauss.html
It contains some of the linear algebra topics.

Reply:
and this one also (https://www.geeksforgeeks.org/linear-algebra/).

Reply:
i'd rather you link a textbook or something of the sort, idt a blog post can have the same amount of rigor

Reply:
Suggest me one. bcz the books that I have do not have a specific chapter about Linear Algebra.

Reply:
Leave it I got a new book. I will do the necessary changes in some minutes.